devbuild.sh

```diff
@@ -502,4 +502,38 @@ else
     fi
 fi
 
+# Copy config/python directories from component to main directory (EE2 compliance)
+if [ "${BUILD_UFS_UTILS}" = "on" ]; then
+  if [ -d "${SRW_DIR}/parm/ufs_utils_parm" ]; then
+    rm -rf ${SRW_DIR}/parm/ufs_utils_parm
+  fi
+  cp -rp ${SRW_DIR}/sorc/UFS_UTILS/parm ${SRW_DIR}/parm/ufs_utils_parm
+fi
+if [ "${BUILD_UPP}" = "on" ]; then
+  if [ -d "${SRW_DIR}/parm/upp_parm" ]; then
+    rm -rf ${SRW_DIR}/parm/upp_parm
+  fi
+  cp -rp ${SRW_DIR}/sorc/UPP/parm upp_parm
```

Comment:
Should this be `cp -rp ${SRW_DIR}/sorc/UPP/parm ${SRW_DIR}/parm/upp_parm`?

```suggestion
  cp -rp ${SRW_DIR}/sorc/UPP/parm ${SRW_DIR}/parm/upp_parm
```

Reply:
You're correct. Updated. Thanks!